deployment-aws/terraform/variables.tf

```diff
@@ -14,6 +14,12 @@ variable "backend_domain" {
   type        = string
 }
 
+variable "batch_instance_ami" {
+  description = "The AMI of the images used for ephemeral Batch instances"
+  type        = string
+  default     = "ami-02202da28664660ac"
```

Comment:
It'd be great to have a brief comment with AMI description (OS, provider, etc). Example: https://github.com/web-mev/mev-backend/blob/9c15a4e7d93161d3c5968d41da3e2c0d1532e180/deployment-aws/terraform/apiserver.tf#L121

Reply:
done